thirdparty/runtime_environments_test.go

```diff
@@ -207,3 +207,33 @@ func TestGetHistory(t *testing.T) {
 	assert.NotEmpty(t, result)
 	assert.Len(result, 15)
 }
+
+func TestGetEvents(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetEvents")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+
+	// Verify that getEvents errors when not provided the required distro field.
+	_, err := c.getEvents(ctx, EventHistoryOptions{})
+	assert.Error(err)
+
+	// Verify that getEvents provides the image events for a distribution.
+	result, err := c.getEvents(ctx, EventHistoryOptions{Image: "ubuntu2204"})
+	require.NoError(t, err)
+	require.NotEmpty(t, result)
```

Comment:
you dont have to require this because nothing depends on results not being empty here

Reply:
require is really nice to use when you don't want tests to panic but one downside is that it will stop the test if the requirements aren't met. 

that's why we only want to use require when it would prevent a potential panic. If you were to refer to `result[0]` in the next line, we would want to use require so that line wouldn't panic. However, since we would want the test to go on even if result in empty in this line, assert would be better

Reply:
same comment for use of require below!